### DISCUSSION TITLE
Controlling POI-Downloads in Cruiser necessary?

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#439] [Q&A] [answered]
Since 3.0.38 I miss the download button - okay, I don't really miss it, because the whole thing now obviously runs under "Map". But I can no longer check the POI downloads there. Do I have to check them at all? Or is it ensured that POIs are also taken into account with every map download and update? BRouter-Tiles can be updated separately in Routing/BRouter, I know.

**Comment by devemux86** [comment, marked as answer]
@WolfgangZ42 thanks for the question.

It depends on what provider we use:

- **Mapsforge**: downloads maps + poi + BRouter data automatically

- **OpenAndroMaps**: downloads contain map + poi

- **Freizeitkarte**: provides only maps

Most users can use the (default) Mapsforge provider and get everything (maps + poi + BRouter data) in 1 place,
the "Offline maps" screen.

> Or is it ensured that POIs are also taken into account with every map download and update?

Correctly, when we download or update the maps, their poi are downloaded or updated automatically too.

> BRouter-Tiles can be updated separately in Routing/BRouter, I know.

BRouter data are updated daily, so it is more important to check them.

**Comment by devemux86** [comment]
The purpose is to make things easier for most users,
who just want to download their region - with one simple action - and start playing with the app.

If you have ideas to make things easier for them, please let me know and we will discuss everything. :slightly_smiling_face:

**Comment by ramondevic** [comment]
There may be users who only want to click on a single button to automatically download maps, points and routing data, but there are other users who want/need to choose what we want to download these files separately, which maps we want and which graphs of routing
This application is very powerful in terms of setting many things and perhaps it will be a step back this scheduled automation without the user being able to choose what they want to download.

I believe that users should be able to choose how we want to update our files: automatically or more manually.

That's what I think.

**Reply by devemux86** [reply]
@ramondevic 

OpenAndroMaps provides zip files that contain map + poi, so both are extracted.

Automation is done with Mapsforge map files which always have corresponding poi files.

Note: GraphHopper graphs remain separate, as they can be activated 1 region each time.

**Reply by ramondevic** [reply]
I've been downloading the files I need from here for a long time.
https://download.mapsforge.org/

Then I manually copy to my phones only what I need.

I hope I can do it like this.

**Reply by devemux86** [reply]
@ramondevic 

Do you use the extra menus to open external maps?

This is always available with the extended menu option (described [here](https://github.com/devemux86/cruiser/discussions/436#discussioncomment-8409758)).

**Comment by WolfgangZ42** [comment]
I have to admit that it was only through https://github.com/devemux86/cruiser/discussions/436#discussioncomment-8409758 that I learned to get the "POI" item in "Map". Even as experienced user you sometimes miss something. So I'm now able to control and update the POIs as I asked for. Together with "Offline Maps" in "Map" and "BRouter" in "Routing" everything is okay for me.
